trakt/interfaces/sync/playback.py

```diff
@@ -16,3 +16,10 @@ def episodes(self, store=None, **kwargs):
             store,
             **kwargs
         )
+
+    @authenticated
+    def delete_progress(self, playbackid, **kwargs):
```

Comment:
Thanks for your comment. I guess we don't need it.
I tried to do that at the very beginning and for some reason
I couldn't find the working solution, but now I got it working.
See latest commit.